src/main/java/duke/SaveFile.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SaveFile {
+    private final String FOLDER_PATH;
+    private final String FILE_PATH;
+    private static File dukeSaveFile;
+
+    /**
+     * Sets the folder path and file path to the new specified path.
+     *
+     * @param folderpath save folder path
+     * @param filepath save file path
+     */
+    public SaveFile(String folderpath, String filepath) {
+        this.FOLDER_PATH = folderpath;
+        this.FILE_PATH = filepath;
+    }
+
+    /**
+     * Creates a save file as a .txt file to store the tasks.
+     */
+    public void createSaveFile() {
+        File saveFolder = new File(FOLDER_PATH);
+        if (!saveFolder.exists() && !saveFolder.isDirectory()) {
+            Duke.printLine("    It looks like u dun have a save folder yet, lemme make one now" + Duke.LS);
```

Comment:
Consider managing all long strings in a separate class called messages

Reply:
Thank you for your feedback. I have refactored them into a separate class